README.md

```diff
@@ -80,15 +80,32 @@ i.e. the CSV dialect specified as:
 }
 ```
 
-Some of the effects of this metadata-free conformance are:
-- The first line of each file must contain the comma-separated list of column names.
-- No comment lines are allowed.
+For the single CSV file of a metadata-free CLDF dataset
+- the first line must contain the comma-separated list of column names,
+- and no comment lines are allowed.
+
+> [!TIP]
+> Thus, a minimal metadata-free CLDF [StructureDataset](modules/StructureDataset) will consist of a CSV
+> file named `values.csv`, with content looking like the example below:
+
+```csv
+ID,Language_ID,Parameter_ID,Value
+1,stan1295,wals-1A,average
+```
 
 
 ### Extended conformance
 
-A dataset is CLDF conformant if it uses a custom metadata file, derived from the default profile for the appropriate 
-module. The metadata MUST specify a `dc:conformsTo` property with one of the CLDF module URIs as value.
+A dataset is CLDF conformant if 
+- it contains a metadata file, derived from the default profile for the appropriate module,
+- at least the minimal set of components (i.e. CSV data files) specified for the module. 
+
+The metadata MUST specify a `dc:conformsTo` property with one of the CLDF module URIs as value.
+
+> [!TIP]
+> Thus, a minimal extended CLDF [StructureDataset](modules/StructureDataset) will consist of
+> - a JSON file containing the metadata (whose name can be freely chosen),
```

Comment:
I'd replace all of the 'whose' referring to file names etc. with something like "a JSON file containing the metadata (with a freely chosen name)".